### ISSUE TITLE
phpstan internal error in console Method newEloquentBuilder() was not found

### ISSUE THREAD

**Issue by expondo** [owen-it/laravel-auditing#808]
| Q                 | A
| ----------------- | ---
| Bug?              | no
| New Feature?      | yes
| Framework         | Laravel
| Framework version | 952.4
| Package version   | 13.5.0
| PHP version       | 8.1
| phpstan/phpstan   | 1.10.6

#### Actual Behaviour

When i use Audit::**query** or SomeModel->audits()->**where**() then phpstan throwin exception
`     Error                                                                                                                                                                                        
 -- ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
     Internal error: Internal error: Method newEloquentBuilder() was not found in reflection of class OwenIt\Auditing\Contracts\Audit. in file AuditService.php
     Run PHPStan with -v option and post the stack trace to:
     https://github.com/phpstan/phpstan/issues/new?template=Bug_report.md
     Child process error (exit code 1):
`
this happen after last release with change model to contract: [release diff](https://github.com/owen-it/laravel-auditing/compare/v13.4.0...v13.5.0)
#### Expected Behaviour

Phpstan has no warning when use builder on Audit model

#### Steps to Reproduce

List **all the steps** needed to reproduce the issue you're having.

Make sure to **include code** (affected models, configuration, ...).

Pastebins, screenshots and/or other resources that may help us understand what's going on, will also be appreciated.

In a nutshell, help us, help you.

#### Possible Solutions

AuditContract should be reveiwed.

**Comment by parallels999** [comment]
@MortenDHansen this was alerady fixed on #807

**Comment by expondo** [comment]
Thanks @parallels999 , 
I replied under #807 because I see, that there is some problem still not fixed.

**Comment by parallels999** [comment]
@expondo thanks, Do you know how to improve this part for phpstan?

**Comment by MortenDHansen** [comment]
Well, phpstan was never a part of the workflow, so i have no idea. Im gonna need to add phpstan to our pipeline. Until then i have no idea what was fixed. Its not an issue i would ever see 🤷

**Comment by expondo** [comment]
@parallels999 not yet.
@erikn69 I can check it tomorrow.

**Comment by fgibaux** [comment]
in my code, I use `$model->audits()->...` method and I'm getting the internal error using this package v13.5.0.
when using dev-master, (so with #807), it removes the phpstan internal error but I get normal errors instead:
```
         Class OwenIt\Auditing\Contracts\OwenIt\Auditing\Contracts\Audit was not found while trying to analyse it - discovering symbols is probably not configured properly.  
         💡 Learn more at https://phpstan.org/user-guide/discovering-symbols                                                                                                  
  17     Class OwenIt\Auditing\Contracts\OwenIt\Auditing\Contracts\Audit was not found while trying to analyse it - discovering symbols is probably not configured properly.  
         💡 Learn more at https://phpstan.org/user-guide/discovering-symbols                                                                                                  
  17     Class OwenIt\Auditing\Contracts\OwenIt\Auditing\Contracts\Audit was not found while trying to analyse it - discovering symbols is probably not configured properly.  
         💡 Learn more at https://phpstan.org/user-guide/discovering-symbols      
```

 I have phpstan v1.10.14, PHP 8.1, Laravel 10.9

**Comment by parallels999** [comment]
@fgibaux try dev branch
https://github.com/owen-it/laravel-auditing/blob/eadb80b096502a52c15560a07ac0554ebc4b1108/composer.json#L67-L69

**Comment by fgibaux** [comment]
@parallels999 ah thank you! it works without errors when using v14-dev

**Comment by parallels999** [comment]
@fgibaux the new version will be released very soon with corrections and performance improvements

**Comment by stefro** [comment]
First of all: Thanks for fixing this! 💪🔥
But it has been a while and there is still no release of these changes yet. Does anyone have an idea when this will be released?
The current release at the time of writing is `13.5.1`.
